src/apiWrappers/carDatabaseApi.ts

```diff
@@ -1,33 +1,73 @@
 import axios from 'axios';
 
-export type CarMake = {
-  Make_ID: number;
-  Make_Name: string;
-};
+//         "objectId": "ElhqsRZDnP",
+// "Year": 2020,
+// "Make": "Cadillac",
+// "Model": "Escalade ESV",
+// "Category": "SUV",
+// "createdAt": "2020-01-27T20:44:17.665Z",
+// "updatedAt": "2020-01-27T20:44:17.665Z"e
+
+export type CarMake = string;
 
-export type CarModel = CarMake & {
-  Model_ID: number;
-  Model_Name: string;
+export type CarModel = {
+  Year: number;
+  Make: string;
+  Model: string;
+  Category: string;
 };
 
-const carDatabaseURL = 'https://vpic.nhtsa.dot.gov/api/vehicles/';
+//  need to set up environments to pass in the variable url string
+const carDatabaseURL =
+  'https://carrectly-admin-staging.herokuapp.com/api/cars/';
 
 export const carDatabaseApi = {
   getAllMakes: async () => {
     const carMakes = await axios
-      .get(`${carDatabaseURL}GetAllMakes?format=json`)
+      .get(`${carDatabaseURL}getAllMakes`)
```

Comment:
I thought we were going to filter the car makes by year as well

Reply:
this is from the admin repo. first we get the unique names of all the car makes. 

then in the second API call we get all the models for a certain year and make


```
router.get('/getAllMakes', async (req, res, next) => {
  try {
    const makes = await CarMakes.findAll({
      attributes: [
        [Sequelize.fn('DISTINCT', Sequelize.col('Make')), 'car_make'],
      ],
    })

    let makesResponse = makes.map((singleBrand) => {
      return singleBrand.dataValues.car_make
    })
    makesResponse.sort((a, b) =>
      a.localeCompare(b, defaultStringCompareOptions)
    )
    res.json(makesResponse)
  } catch (err) {
    next(err)
  }
})

router.get('/getModelsByYearMake/:make/:year', async (req, res, next) => {
  try {
    const models = await CarMakes.findAll({
      where: {
        [Op.and]: [{ Year: req.params.year }, { Make: req.params.make }],
      },
    })
    res.json(models)
  } catch (err) {
    next(err)
  }
})
```